events/pendingfolderschanged.rst

```diff
@@ -6,8 +6,10 @@ PendingFoldersChanged
 .. versionadded:: 1.14.0
 
 Emitted when pending folders were added / updated (offered by some
-device, but not shared to them) or removed (folder ignored or added or
-no longer offered from the remote device).
+device, but not shared to them) or removed (folder ignored, dismissed
+or added or no longer offered from the remote device).  An entry
+without a ``device`` attribute means that the folder is no longer
+pending for any device.
```

Comment:
What is this about? I.e. why is the device removed respectively why is it any different than the existing "remove-events"?

Reply:
This is a shortcut for when a pending folder is ignored locally.  Any pending entries with that folder ID should then be discarded, so instead of enumerating them all, only one entry with the `folderID` attribute is emitted.  That is handled in the GUI by removing a whole nested sub-object.